src/models/team.js

```diff
@@ -13,7 +13,18 @@ const team = new Schema({
       addDateTime: { type: Date, default: Date.now(), ref: 'userProfile' },
     },
   ],
-  teamCode: { type: 'String', default: '' },
+  teamCode: {
```

Comment:
Nicely done.

I would highly recommend that the createdDatetime property in the team Schema, should have a default value of `Date.now()`

Reply:
Good suggestion. I've added a default value for createdDatetime, please check.

Reply:
Acknowledged.

Well done.